src/index.js

```diff
@@ -12,13 +12,15 @@ const { customOptionsHook } = require('./hooks/custom-options');
 const { browserHeadersHook } = require('./hooks/browser-headers');
 const { proxyHook } = require('./hooks/proxy');
 const { http2Hook } = require('./hooks/http2');
+const { insecureParserHook } = require('./hooks/insecure-parser');
 
 const gotScraping = got.extend({
     mutableDefaults: true,
     ...SCRAPING_DEFAULT_OPTIONS,
     context: {
         headerGenerator: new HeaderGenerator(),
         useHeaderGenerator: true,
+        insecureParser: true,
```

Comment:
We should also validate the option in the `optionsValidationHandler`

Reply:
Actually it's there